compiler/sources.go

```diff
@@ -0,0 +1,115 @@
+package compiler
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+	"sort"
+
+	"github.com/neelance/astrewrite"
+)
+
+// sources is a slice of parsed Go sources.
```

Comment:
I guess these are all source files in a specific package, given a single ImportPath? Is it worth mentioning that in the comment?

Reply:
This is a bit tricky... It is true that all sources should belong to the same package, they don't have to represent a real package that exists on the file system (e.g. playground) or may be only a part of a package (e.g. `gopherjs build foo.go bar.go`). I'll try to add this to the comment.